### ISSUE TITLE
Remote Desktop feature in v22.04 cannot be permanently turned off once enabled

### ISSUE THREAD

**Issue by wagnerck** [pop-os/gnome-control-center#230]
A customer reported, and I have replicated on a test system, that once you enable the Remote Desktop feature in the Sharing panel of the Settings app, that you can no longer keep it turned off. If you disable it through the Settings app, it will turn back on when you reboot.

This happens regardless of the combination of switch settings in the Sharing panel. If you disable the Remote Desktop feature alone, it comes back after reboot; if you disable the whole Sharing panel with the switch at the top, it comes back; if you disable the RDP feature and _then_ the whole Sharing panel, it comes back.

Running these commands will temporarily turn the RDP service off, same as in the Settings app:
```
systemctl --user stop gnome-remote-desktop.service
systemctl --user disable gnome-remote-desktop.service
```
but it will be re-enabled on reboot. The following additional command will prevent it from coming back after reboot, although it may still show as enabled in Settings:
```
systemctl --user mask gnome-remote-desktop.service
```
I've provided the `mask` option as a temporary workaround for a customer. Unmasking the service and then rebooting causes it to automatically enable itself again, even if it's showing as off in the Settings app before the reboot.

This appears to be a problem inherited from upstream Ubuntu (https://askubuntu.com/questions/1404631/why-are-the-options-sharing-and-remote-desktop-always-on-between-restarts-on-ubu) and the `gnome-remote-desktop` package does not have a Pop!OS fork or build:
```
ckw@flatline:~$ apt policy gnome-remote-desktop 
gnome-remote-desktop:
  Installed: 42.0-4ubuntu1
  Candidate: 42.0-4ubuntu1
  Version table:
 *** 42.0-4ubuntu1 500
        500 http://us.archive.ubuntu.com/ubuntu jammy/main amd64 Packages
        100 /var/lib/dpkg/status
```

I believe that the `systemctl disable` and `enable` functions for the service are broken in some way but I'm not 100% sure about that.

**Comment by wagnerck** [comment]
This appears to have been resolved upstream. Package version `42.1.1-0ubuntu1` was installed through the usual update process this morning, and here's the changelog:
```
gnome-remote-desktop (42.1.1-0ubuntu1) jammy; urgency=medium

  * New upstream release (LP: #1970662)
    - Fixes black screen with virtio on qemu (LP: #1971195)
  * Drop all patches: applied in new release
  * Depend on libmutter instead of gnome-shell | budgie-desktop
    - This is a more accurate dependency
  * Require libmutter 42.1 for Nvidia fixes
  * Depend on fuse3 (Closes: #998846) (LP: #1970411)
  * Don't automatically enable the systemd user service (LP: #1973028)
  * Add postinst to remove the automatic enabling of the user service

 -- Jeremy Bicha <jbicha@ubuntu.com>  Tue, 17 May 2022 14:27:16 -0400
```
Note those last two. I was able to successfully disable the remote desktop feature using the Sharing panel in the Settings app, and it was still disabled on reboot.
```
ckw@flatline:~$ systemctl --user status gnome-remote-desktop.service 
○ gnome-remote-desktop.service - GNOME Remote Desktop
     Loaded: loaded (/usr/lib/systemd/user/gnome-remote-desktop.service; disabl>
     Active: inactive (dead)
```
This is worth testing to make sure the fix is in place across multiple systems but it's working correctly now for me.

**Comment by leviport** [comment]
I tested on a couple machines, and I'm also able to turn off Remote Desktop. I'll close this for now.
